Core/Core/Views/AvatarView.swift

```diff
@@ -36,11 +36,13 @@ open class AvatarView: UIView {
 
             addSubview(label)
             label.pin(inside: self)
+            label.allowsDefaultTighteningForTruncation = true
             label.backgroundColor = .named(.backgroundLightest)
             label.clipsToBounds = true
             label.isAccessibilityElement = false
             label.layer.borderColor = UIColor.named(.borderMedium).cgColor
             label.layer.borderWidth = 1 / UIScreen.main.scale
+            label.lineBreakMode = .byClipping
```

Comment:
I was seeing a 3 character initials getting an ellipsis, hence the changes in this file. We should only have 2 characters max, & should never see an ellipsis.